interface/weno/logsync.php

```diff
@@ -0,0 +1,119 @@
+<?php
+
+/*
+ *  @package OpenEMR
+ *  @link    http://www.open-emr.org
+ *  @author  Sherwin Gaddis <sherwingaddis@gmail.com>
+ *  @copyright Copyright (c) 2021 Sherwin Gaddis <sherwingaddis@gmail.com>
+ *  @license https://github.com/openemr/openemr/blob/master/LICENSE GNU General Public License 3
+ */
+
+require_once("../globals.php");
+require_once($GLOBALS["srcdir"] . "/formatting.inc.php");
+
+use OpenEMR\Common\Logging\EventAuditLogger;
+use OpenEMR\Rx\Weno\Container;
+
+//TODO convert this entire file to a class
+$container = new Container();
+
+$wenoProperties = $container->getTransmitproperties();
+$provider_info = $wenoProperties->getProviderEmail();
+$logsync = $container->getLogproperties();
+
+/**
+ * checks to see if the file exist and if it does was it put there today?
+ * The idea behind this is to automate the log information download and import into the database.
+ * This should only execute once a day no matter how many times it is called.
+ * The idea was to include in the index file to be executed when the prescription called.
+ */
+$today = date("F d Y");
+$filedate = $logsync->doesLogFileExist();
+//die if the dates match or the file does not exist
+if ($today !== $filedate) {
+    $logurlparam = $logsync->logEcps();
+    $syncLogs = "https://online.wenoexchange.com/en/EPCS/DownloadNewRxSyncDataVal?useremail=";
+    if ($logurlparam == 'error') {
+        echo xlt("Cipher failure check encryption key");
+        exit;
+    }
+
+    $urlOut = $syncLogs . $provider_info['email'] . "&data=" . urlencode($logurlparam);
+
+    $ch = curl_init($urlOut);
+    curl_setopt($ch, CURLOPT_TIMEOUT, 200);
+    curl_setopt($ch, CURLOPT_RETURNTRANSFER, 1);
+    $rpt = curl_exec($ch);
+    if (curl_errno($ch)) {
+        throw new Exception(curl_error($ch));
+    }
+    $statusCode = curl_getinfo($ch, CURLINFO_HTTP_CODE);
+    curl_close($ch);
+    if ($statusCode == 200) {
+        file_put_contents($logsync->rxsynclog, $rpt);
+        $logstring = "prescrition log import initiated successfully";
```

Comment:
mispell here, should be:
```
prescription log import initiated successfully
```

Reply:
i also noted 2 other places in codebase in src/Rx/Weno/LogProperties.php where this same string is misspelled. Should that also be fixed?

Reply:
Or should we keep these misspellings (ie. is there something dependent on this string upstream or downstream)?